test/integration/CustomsNotificationMetricsConnectorSpec.scala

```diff
@@ -79,31 +79,31 @@ with BeforeAndAfterAll with CustomsNotificationMetricsService with AuditService
       intercept[RuntimeException](await(sendValidRequest())).getCause.getClass shouldBe classOf[NotFoundException]
```

Comment:
blank line after `intercept ..` pls to preserve `GIVEN/WHEN/THEN` block formatting and also for the other tests